### ISSUE TITLE
I can't visualize the following bvh

### ISSUE THREAD

**Issue by vittorione94** [facebookresearch/fairmotion#58]
[TEST_clip.bvh.zip](https://github.com/facebookresearch/fairmotion/files/7718999/TEST_clip.bvh.zip)

Hello,

First of all thanks for this repo! I've tried to visualize this bvh file but without success. I've tried translating it and scaling it but I still wasn't able to visualize it correctly. However if I import the file in Blender it works perfectly. 
Any help would be very much appreciated :) 

Thanks for your time.
-Vittorio

**Comment by dpacgopinath** [comment]
Hi Vittorio, I made a few observations to visualize the motion:
1. The visualizer interprets measurements in meters. Since the magnitude of positions and offsets were in double/triple digits, I had to use the `--scale` argument to scale the character down in size
2. I also had to increase the thickness of the character as a result (using `--thickness`)
3. The motion has y-axis as the up vector, so we'll have to use the `--axis-up` argument

So, the following command should help you visualize the attached test motion. I have attached a screenshot.
```
python fairmotion/viz/bvh_visualizer.py --bvh-files TEST_clip.bvh --thickness 20.0 --scale 0.005 --axis-up y
```
<img width="659" alt="Screen Shot 2021-12-15 at 2 03 51 PM" src="https://user-images.githubusercontent.com/1420491/146249856-c95040be-2271-4f10-a1f0-bc52de061b20.png">
